core/src/backends/eclair/rest/node.rs

```diff
@@ -119,4 +121,17 @@ impl NodeMethods for EclairRest {
 
         Ok(data.try_into()?)
     }
+
+    async fn get_invoice(&self, payment_hash: String) -> Result<Invoice, Error> {
+        let url = format!("{}/getreceivedinfo", self.config.url);
+
+        let mut params = HashMap::new();
+        params.insert("paymentHash", &payment_hash);
+
+        let mut response = self.client.post(&url).form(&params).send().await?;
+        response = Self::on_response(response).await?;
+        let data: InvoiceResponse = response.json().await?;
```

Comment:
Not really important, but we added new lines between these in every other methods.
```suggestion
        let mut response = self.client.post(&url).form(&params).send().await?;
        
        response = Self::on_response(response).await?;
        
        let data: InvoiceResponse = response.json().await?;
```